lib/widgets/booking/booking_card.dart

```diff
@@ -36,6 +38,21 @@ class BookingCard extends StatelessWidget {
           duration: const Duration(milliseconds: 300),
           curve: Curves.easeInOutCubic,
           builder: (context, value, child) {
+            final gradientAlignmentX = -0.4;
+            final gradientAlignmentY = -0.2;
+            final gradientStops = [
+              0.0.toDouble(),
+              0.5.toDouble(),
+              0.5.toDouble(),
+              1.toDouble()
+            ];
+            final gradientColors = [
+              const Color.fromARGB(16, 0, 0, 0),
+              const Color.fromARGB(16, 0, 0, 0),
+              Colors.white10,
+              Colors.white10,
+            ];
```

Comment:
These gradient variables only make sense when `BookingCard.isDisabled`, so they should only be computed in that case. Maybe by moving their declaration straight into their usage—less cleaner way, but more performant (?):

```dart
decoration: (!isDisabled)
    ? BoxDecoration(
        color: Theme.of(context)
            .cardColor
            .withOpacity(isBeforeNow ? 0.41 : 1.0),
        borderRadius: BorderRadius.circular(10),
      )
    : BoxDecoration(
        borderRadius: BorderRadius.circular(10),
        gradient: LinearGradient(
          begin: Alignment.topLeft,
          end: const Alignment(-0.4, -0.2),
          stops: const [0.0, 0.5, 0.5, 1.0],
          colors: [
            const Color.fromARGB(16, 0, 0, 0),
            const Color.fromARGB(16, 0, 0, 0),
            Colors.white10,
            Colors.white10,
          ],
          tileMode: TileMode.repeated,
        ),
      ),
```

Reply:
Yes, I did it this way to get rid of `Prefer const with constant contructors`and 
`Prefer const literals as parameters of constructors on @inmutable classes` linting warnings. But I agree with you that it doesn't make much sense in this case.

Changing code to your suggestion 😉